GS01SUMO/GS01SUMO_R00003_ingest.csv

```diff
@@ -0,0 +1,79 @@
+mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/cg_data/dcl11/imm/ctdmo04/*.DAT,GS01SUMO-RII11-02-CTDMOQ014,recovered_host,Available,
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/instruments/dcl11/imm/CTDMO_sn37-11475/*.hex,GS01SUMO-RII11-02-CTDMOQ014,recovered_inst,Available,
+mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/cg_data/dcl11/imm/ctdmo05/*.DAT,GS01SUMO-RII11-02-CTDMOQ015,recovered_host,Available,
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/instruments/dcl11/imm/CTDMO_sn37-13602/*.hex,GS01SUMO-RII11-02-CTDMOQ015,recovered_inst,Available,
+mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/cg_data/dcl11/imm/ctdmo06/*.DAT,GS01SUMO-RII11-02-CTDMOQ016,recovered_host,Available,
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/instruments/dcl11/imm/CTDMO_sn37-13615/*.hex,GS01SUMO-RII11-02-CTDMOQ016,recovered_inst,Available,
+mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/cg_data/dcl11/imm/ctdmo07/*.DAT,GS01SUMO-RII11-02-CTDMOQ017,recovered_host,Available,
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/instruments/dcl11/imm/CTDMO_sn37-13619/*.hex,GS01SUMO-RII11-02-CTDMOQ017,recovered_inst,Available,
+mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/cg_data/dcl11/imm/ctdmo08/*.DAT,GS01SUMO-RII11-02-CTDMOR018,recovered_host,Available,
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/instruments/dcl11/imm/CTDMO_sn37-12566/*.hex,GS01SUMO-RII11-02-CTDMOR018,recovered_inst,Available,
+mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/cg_data/dcl11/imm/ctdmo09/*.DAT,GS01SUMO-RII11-02-CTDMOR019,recovered_host,Available,
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/instruments/dcl11/imm/CTDMO_sn37-12649/*.hex,GS01SUMO-RII11-02-CTDMOR019,recovered_inst,Available,
+mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/cg_data/dcl11/imm/ctdmo10/*.DAT,GS01SUMO-RII11-02-CTDMOR020,recovered_host,Available,
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/instruments/dcl11/imm/CTDMO_sn37-13651/*.hex,GS01SUMO-RII11-02-CTDMOR020,recovered_inst,Available,
+#mi.dataset.driver.dosta_abcdjm.ctdbp_p.dosta_abcdjm_ctdbp_p_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/instruments/dcl11/imm/CTDBP_sn16-50159/SBE*.hex,GS01SUMO-RII11-02-DOSTAD031,recovered_inst,Not Expected,No data available

```

Comment:
Same comment as above for this CTDBP

Reply:
According to the data recovery checklist "Instrument: 137MB XML file. Download stopped at ~3%. No HEX file due to incomplete download. Time stamps all over and excessive amount of data (later parts all same HEX string)."